inc/views/base/usercp/subscribe_thread.twig

```diff
@@ -0,0 +1,57 @@
+{% extends 'layouts/master.twig' %}
+
+{% block head %}
+    <title>{{ mybb.settings.bbname }} - {{ lang.forum_subscriptions }}</title>
+{% endblock head %}
+
+{% block body %}
+    <form action="usercp2.php" method="post" name="input">
```

Comment:
Shouldn't this action be `usercp.php` instead since we've removed `usercp2.php`?

Reply:
Yup, I'll update it in a moment.